content/zh/docs/reference/config-api/client-authentication.v1beta1.md

```diff
@@ -180,10 +241,16 @@ the transport.
 <a href="#client-authentication-k8s-io-v1beta1-Cluster"><code>Cluster</code></a>
 </td>
 <td>
+<!-- 
    Cluster contains information to allow an exec plugin to communicate with the
 kubernetes cluster being authenticated to. Note that Cluster is non-nil only
 when provideClusterInfo is set to true in the exec provider config (i.e.,
-ExecConfig.ProvideClusterInfo).</td>
+ExecConfig.ProvideClusterInfo).
+-->
+   此字段中包含的信息使得 exec 插件能够与要访问的 Kubernetes 集群通信。
+   注意，cluster 字段只有在 exec 驱动的配置中 provideClusterInfo
```

Comment:
```suggestion
   注意，cluster 字段只有在 exec 驱动的配置中 provideClusterInfo
```
Referring to other Chinese localizations, the proper noun `cluster` is usually translated

Reply:
the cluster is a field name of a struct

Reply:
thank,I get it